cmd/generate/config/main.go

```diff
@@ -14,179 +14,188 @@ const (
 	templatePath = "rules/config.tmpl"
 )
 
+//go:generate go run $GOFILE ../../../config/gitleaks.toml
```

Comment:
`$GOFILE` implies the current file `main.go`. See [variables](https://pkg.go.dev/cmd/go/internal/generate).